libraries/protocol/steem_operations.cpp

```diff
@@ -318,6 +318,13 @@ namespace golos { namespace protocol {
             });
         }
 
+        void chain_properties_21::validate() const {
+            chain_properties_19::validate();
+            GOLOS_CHECK_VALUE_LEGE(worker_from_reward_fund_percent, 0, STEEMIT_100_PERCENT);
+            GOLOS_CHECK_VALUE_LEGE(worker_from_vesting_fund_percent, 0, STEEMIT_100_PERCENT);
+            GOLOS_CHECK_VALUE_LEGE(worker_from_witness_fund_percent, 0, STEEMIT_100_PERCENT);
```

Comment:
values are unsigned, so GE check is always true

Reply:
@zxcat But GOLOS_CHECK_VALUE_LEGE is everywhere used to check percents in steem_operations.cpp